DatadogCore/Tests/Datadog/DatadogCore/DatadogCoreTests.swift

```diff
@@ -45,7 +45,6 @@ class DatadogCoreTests: XCTestCase {
             applicationVersion: .mockAny(),
             backgroundTasksEnabled: .mockAny()
         )
-        defer { core.flushAndTearDown() }
```

Comment:
I can see a new test for stopping, but I'm wondering if we need removing these?

It seems like it was here to clean things up after each test.
Do you know the exact reason of having them? Maybe a way to go is to `defer { core.stop() }` now?

Reply:
This was a duplicate call, the core is flushed before assertion 👍